src/Interpreter.h

```diff
@@ -116,6 +116,14 @@ class Interpreter {
         delete &rel;
     }
 
+    /** Swap relation */
+    void swapRelation(const RamRelation& ramRel1, const RamRelation& ramRel2) {
+        InterpreterRelation* rel1 = &getRelation(ramRel1);
```

Comment:
Can this be done with `std::swap(getRelation(ramRel1), getRelation(ramRel2));`?
Or for better speed at the expensive of verbosity, `std::swap(environment[ramRel1.getName()], environment[ramRel2.getName()]);`

Reply:
Not sure about this - perhaps we discuss this offline.